Source/Core/TaskProcessor.js

```diff
@@ -3,7 +3,7 @@ import when from "../ThirdParty/when.js";
 import buildModuleUrl from "./buildModuleUrl.js";
 import defaultValue from "./defaultValue.js";
 import defined from "./defined.js";
-import destroyObject from "./destroyObject.js";
+// import destroyObject from "./destroyObject.js";
```

Comment:
If this import is not being used, it should be deleted not commented out. What is the reason behind not using `destroyObject`?

Reply:
because `destroyObject` does not destroy properly 
you can llok at the `destroyObject` method its only delete functions, actually I did not understand why only remove functions.
it can be an array or an object with big amount of data right ?
I dont know why they do this way.  
So I remove it and I remove everything in object